code/modules/organs/pain.dm

```diff
@@ -29,23 +29,41 @@ mob/var/next_pain_time = 0
 	if(burning)
 		switch(amount)
 			if(1 to 10)
-				msg = "<span class='danger'>Your [partname] burns.</span>"
+				if(species.flags & IS_PLANT)
+					msg = "<b>The nymph making up our [partname] notes a burning injury.</b>"
+				else
+					msg = "<span class='danger'>Your [partname] burns.</span>"
```

Comment:
These overrides are really icky. There has to be a better way to do this. In case we add different species the flag-checking just won't cut it here.